netlify.toml

```diff
@@ -4,3 +4,28 @@ name = "snooty-cache-plugin"
 [build]
 publish = "snooty/public"
 command = ". ./build.sh"
+
+# Redirect calculated fields to new drawer introduced in DOCSP-32370 - Field Customizations
+# Field Customization
+[[redirects]]
+from = "/mapping-rules/calculated-fields/*"
+to = "/mapping-rules/fields/calculated-fields/:splat"
+
+# Redirect for previous Code Generation Page DOCSP-33055 - Prep Code Generation
+[[redirects]]
+from = "/code-generation/code-generation"
+to = "/code-generation"
+
+# DOCS-16498 Getting Started Page Start
+[[redirects]]
+from = "/migration-scenarios"
+to = "/getting-started/migration-scenarios"
+
+[[redirects]]
+from = "/supported-databases"
+to = "/getting-started/supported-databases"
+
+[[redirects]]
+from = "/connection-strings/connection-strings"
+to = "/release-notes"
+force = true
```

Comment:
I think this may be left over from our intial testing session, we should remove this one :)

```suggestion

```

Reply:
Good catch, thank you!